src/components/BackgroundContainer.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useState } from 'react'
+// @ts-expect-error -- TODO: type useCustomWallpaper
```

Comment:
Do we have some card or reminder to handle this TODO later? Or should it be handled in this PR?

Reply:
It should be another PR in my mind